03.bowling/bowling.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+point = frames.each.with_index.sum do |frame, i|
+  next_frame = frames[i + 1]
+  frame_after_next = frames[i + 2]
+  if i < 9
+    if frame[0] == 10
+      if next_frame[0] == 10
+        10 + frame_after_next[0] + frame.sum
```

Comment:
この10は`next_frame[0]`の10だと思います。
ここだけリテラルの10が登場するのは一貫性がないので、

```suggestion
        next_frame[0] + frame_after_next[0] + frame.sum
```

みたいになってる方がいいかなと思いました。（frame, next_frame, frame_after_next、の順で並ぶとより自然かも？）

Reply:
ストライクだからまずは10を足して……という感じで書いていましたが、確かに一貫性がなく可読性が低いと思いました🙇‍♂️
修正し、ループ内でのスコアの加算の順番も frame, next_frame, frame_after_next になるよう、その他の箇所も修正しております。https://github.com/junohm410/ruby-practices/pull/3/commits/f6ff1b75f8b03f05e7bbe4e9f1d8e92e3edf4ea2